queries.sql

```diff
@@ -19,3 +19,23 @@ AND '2019-01-01';
 
 SELECT * FROM animals WHERE name LIKE '%mon';
 
+
```

Comment:
- [ ] I believe you are missing some requirements for this file. If you read carefully, you will see that you need to apply some queries in-between transactions, so please make sure to add the code for it. Here is an image to clarify: 
![Captura de pantalla 2022-07-09 155459](https://user-images.githubusercontent.com/59100477/178108895-987ea100-a31f-4cae-af05-a103973bc96b.jpg)